cl/favorites/templates/user_prayers.html

```diff
@@ -0,0 +1,145 @@
+{% extends "base.html" %}
+{% load extras %}
+{% load text_filters %}
+{% load static %}
+{% load pacer %}
+{% load tz %}
+{% load humanize %}
+
+{% block title %}{% if is_page_owner %}Your RECAP Requests {% else %}RECAP Requests for: {{ requested_user }}{% endif %} – CourtListener.com{% endblock %}
+{% block og_title %}{% if is_page_owner %}Your RECAP Requests {% else %}RECAP Requests for: {{ requested_user }}{% endif %} – CourtListener.com{% endblock %}
+{% block description %}CourtListener lets you request purchase of legal documents. View the document requests for {{ requested_user }}.{% endblock %}
+{% block og_description %}CourtListener lets you request purchase of legal documents. View the document requests for {{ requested_user }}.{% endblock %}
+
+{% block content %}
+  <div class="col-xs-12">
+    <h1 class="text-center">{% if is_page_owner %}Your RECAP Requests {% else %}RECAP Requests for: {{ requested_user }}{% endif %}</h1>
+  </div>
+
+  <div class="col-xs-12" id="prayer_summary" hx-swap-oob="true"
+    {% if request.user.is_authenticated %}
+    hx-get="{% url 'user_prayers' request.user.username %}"
+    hx-trigger="prayersListChanged from:body" hx-swap="none"
+    {%endif %}
+  >
+    <div class="well well-sm">
+      {% if is_page_owner %}
+      <p>
+        You have had <b>{{ count|intcomma }}</b> requests fulfilled for documents that cost <b>${{total_cost|floatformat:2 }}</b>.
+      </p>
+      <p>
+        {% if is_eligible %}You are eligible to make document requests.{% else %}You have reached your daily limit; wait 24 hours to make new requests.{% endif %}
+      </p>
+      {% endif %}
+    </div>
+  </div>
+
+  <div class="col-xs-12" id="prayer_list" hx-swap-oob="true">
+    <div class="table-responsive">
+      <table class="settings-table table">
+        <thead>
+            <tr>
+              <th>Court</th>
+              <th>Case Name</th>
+              <th>Document Number</th>
+              <th>Document Description</th>
+              <th>Requested On</th>
+              <th>Status</th>
+              {% if is_page_owner %}<th>Action</th>{% endif %}
+            </tr>
+        </thead>
+        <tbody>
+            {% for prayer in prayers %}
+            <tr>
+              <td>{{ prayer.docket_entry.docket.court.citation_string }}</td>
+              {% with docket=prayer.docket_entry.docket %}
+              <td>
+                <a href="{% url "view_docket" docket.pk docket.slug %}">
+                  {{ prayer.docket_entry.docket|best_case_name|safe|v_wrapper }} ({{ prayer.docket_entry.docket.docket_number }})
```

Comment:
We can simplify this line to:

```suggestion
                   {{ docket|best_case_name|safe|v_wrapper }} ({{ docket.docket_number }})
```

Reply:
You're right! Thanks for catching this.